torch/_inductor/ir.py

```diff
@@ -5430,7 +5428,21 @@ def require_channels_last_3d(cls, x):  # type: ignore[no-untyped-def]
 
     @classmethod
     def require_contiguous(cls, x):  # type: ignore[no-untyped-def]
-        return cls.require_stride_order(x, list(reversed(range(len(x.get_size())))))
+        def safe_get_name(x):
```

Comment:
Nit: I still would prefer that we fix this on the callee side. Can you add a TODO ?

Reply:
Added TODO.